jekyll/_cci2/container-runner.adoc

```diff
@@ -475,89 +475,30 @@ While jobs that run with container-agent cannot use CircleCI's <<building-docker
 
 You can see an example link:https://discuss.circleci.com/t/setup-remote-docker-on-container-runner/45629/11?u=sebastian-lerner[on our community forum] of how some users have successfully used kaniko to build a container image.
 
-Another option is to use a tool called link:https://github.com/containers/buildah[Buildah]. Buildah can be used in your `.circleci/config.yml` syntax:
-
-```yaml
-docker:
-  - image: quay.io/buildah/stable:v1.27.0
-```
-
-[#using-the-buildah-image]
-=== Using the Buildah image
-
-Buildah relies on the link:https://github.com/containers/fuse-overlayfs[fuse-overlay] program inside of the container, which means that a fuse device plugin must be configured in order to use it. `/dev/fuse` is required to use `fuse-overlayfs` inside of the container, as this option tells Buildah on the host to add `/dev/fuse` to the container for Buildah's use. Kubernetes has a device plugin system to enable secure sharing of host devices with pods.
-
-To install the configuration `dev/fuse`, clone this link:https://github.com/kuberenetes-learning-group/fuse-device-plugin/blob/master/fuse-device-plugin-k8s-1.16.yml[repository] to where you are running Helm commands for your container-agent deployment. Then run:
-
-```
-kubectl create -f fuse-device-plugin-k8s-1.16.yml
-```
-
-You can confirm that this has been configured correctly by running `kubectl get daemonset -n kube-system` and confirming that `fuse-device-plugin-daemonset` is present and ready.
-
-Once this device has been added, update the container-agent <<#resource-class-configuration-custom-pod,resource class configuration>>:
+Another option is to use a tool called link:https://github.com/nestybox/sysbox[Sysbox] which must first be link:https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md[installed on your Kuberenetes cluster]. 
+Once installed, update the container-agent <<#resource-class-configuration-custom-pod,resource class configuration>> and apply the changes:
 
 ```yaml
 resourceClasses:
  <namespace>/<resourceClass>:
   token: <token>
-   spec:
-    containers:
-     - resources:
-        limits:
-         github.com/fuse: 1
+  metadata:
```

Comment:
<img width="925" alt="Screenshot 2025-01-23 at 16 38 02" src="https://github.com/user-attachments/assets/b11831f0-58ae-4d1f-8a22-49caf407d4ea" />
 Is this indentation correct?

Reply:
yes it is!